blocks/job-list/job-list.js

```diff
@@ -0,0 +1,59 @@
+export default function decorate(block) {
+  const jobList = document.createElement('div');
+  jobList.className = 'job-list';
+
+  const [departmentRow, ...jobRows] = [...block.children];
+  
+  if (departmentRow) {
+    const departmentTitle = document.createElement('h2');
+    departmentTitle.className = 'job-list__department util-heading-s';
+    departmentTitle.textContent = departmentRow.textContent;
+    jobList.append(departmentTitle);
+  }
+
+  const jobListings = document.createElement('div');
+  jobListings.className = 'job-list__listings';
+
+  jobRows.forEach(row => {
+    const [title, department, location, jobId] = row.children;
+    
+    const jobListing = document.createElement('a');
+    jobListing.className = 'job-listing';
+    
+    const contentWrapper = document.createElement('div');
+    contentWrapper.className = 'job-listing__content';
+    
+    if (title) {
+      const titleE = document.createElement('h3');
```

Comment:
`<a>` elements cannot contain semantic elements like `h3`, `p`, etc. We'll need to use spans. I've made a little helper function out of some code @arnest00 wrote for callout, [it's on the section header PR,](https://github.com/adobe/adobe-design-website/pull/71/files#diff-4f36d2c1c31c44a34f0e4209e07ce030c08a4bf8f03cb3461118be9daff81a40) it might be helpful here!

Reply:
Okay! I updated these to spans manually for now until this get merged it was low effort!